aws/resource_aws_sfn_state_machine_test.go

```diff
@@ -119,7 +163,7 @@ func testAccCheckAWSSfnStateMachineDestroy(s *terraform.State) error {
 		})
 
 		if err != nil {
-			if wserr, ok := err.(awserr.Error); ok && wserr.Code() == "StateMachineDoesNotExist" {
+			if isAWSErr(err, sfn.ErrCodeStateMachineDoesNotExist, "") {
 				return nil
```

Comment:
Nit: Should `continue` in `CheckDestroy` functions to verify multiple resources

```suggestion
				continue
```